Code Cleanup

nathanleiby opened the pull request:
See commits:

- removing unused 3rd party code
- reorganizing some files

nathanleiby committed:
remove SuperTiled2Unity

nathanleiby committed:
move LevelSelect script

nathanleiby committed:
move files out of Assets rootdir

peternga commented:
Weird, I thought I removed SuperTiled2Unity when I switched to Unity's tile editor. 🤷‍♂

nathanleiby commented:
Not sure! I'm wondering if git lfs is giving us issues... :/

On Sat, Apr 4, 2020, 17:59 peternga <notifications@github.com> wrote:

> Weird, I thought I removed SuperTiled2Unity when I switched to Unity's
> tile editor. 🤷‍♂
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/CleverGameDev/Cleverfall/pull/15#issuecomment-609111646>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAY6YW72YWHRD55S6KVGALRK7JXRANCNFSM4L7KVUWQ>
> .
>
